roles/gitops_configure_repo/README.md

```diff
@@ -7,15 +7,15 @@ Requirements
 ------------
 
 * ArgoCD/GitOps already configured
-* The repository MUST be hosted in Gitlab
+* The repository MUST grant access through SSH keys.
 * The SSH key has permissions to read from Gitlab repository.
```

Comment:
Don't think we need the reference to gitlab here.

Reply:
Thanks for the comment. I removed the reference to gitlab.